src/js/components/help/historyPage.jsx

```diff
@@ -20,39 +20,44 @@ export default class HelpPage extends React.Component {
 
         this.state = {
             history: '',
-			sections: []
+			sections: [],
```

Comment:
if we're only using clSections and tSections we don't really need the regular sections state